server.js

```diff
@@ -43,6 +55,6 @@ app.listen(port, '0.0.0.0', function onStart(err) {
   if (err) {
     console.log(err);
   }
-
+  console.info('Node environment is set to: %s', process.env.NODE_ENV);
   console.info('==> Listening on port %s. Open up http://0.0.0.0:%s/ in your browser.', port, port);
```

Comment:
I don't actually think 0.0.0.0 is allowed on the back end. Switch to "localhost"?